toolkit/cli/smart-contracts-commands/src/register.rs

```diff
@@ -0,0 +1,60 @@
+use jsonrpsee::http_client::HttpClient;
+use partner_chains_cardano_offchain::{await_tx::FixedDelayRetries, register::run_register};
+use sidechain_domain::{
+	AdaBasedStaking, AuraPublicKey, BlockProducerRegistration, GrandpaPublicKey,
+	MainchainPublicKey, MainchainSignature, SidechainPublicKey, SidechainSignature, UtxoId,
+};
+
+use crate::{parse_sidechain_public_keys, read_private_key_from_file};
+
+#[derive(Clone, Debug, clap::Parser)]
+pub struct RegisterCmd {
+	#[clap(flatten)]
+	common_arguments: crate::CommonArguments,
+	#[arg(long)]
+	genesis_utxo: UtxoId,
+	#[arg(long)]
+	registration_utxo: UtxoId,
+	#[arg(long)]
+	payment_key_file: String,
+	#[arg(long)]
+	stake_signing_key_file: String,
```

Comment:
I think this should not be present here?

Reply:
Indeed, thank you!